api/configserver/save.go

```diff
@@ -212,6 +214,13 @@ func saveConfigRequestUnmarshler(r *http.Request) (atc.Config, db.PipelinePaused
 				}
 			}
 
+			if dataKind == reflect.String {
```

Comment:
is there any way to do this only if the valKind expects a `time.Duration` / `atc.Duration`?